### ISSUE TITLE
[BUG] Couldn't run tests using Playwright on M1 Macbook along with Docker

### ISSUE THREAD

**Issue by chanagonda** [microsoft/playwright#11683]
**Context:**
- Playwright Version: [1.16.3]
- Operating System: [Mac,  M1 Pro Chip]
- Node.js version: [16.10.0]
- Browser: [  Chromium ( basically All browsers )]

System:

- OS: Mac v 12.1, Apple M1 Pro
- Memory: 16 GB
- Docker locally installed

Binaries:

- Node: 16.0.0 - ~/n/bin/node
- npm: 7.24.0 - ~/n/bin/npm
- playwright - 1.16.3
 

**Code Snippet**

Jest.config.js file

```
 testEnvironmentOptions: {
    'jest-playwright': {
      browsers: [
        {
          browserType: 'chromium',
          args: ['--disable-dev-shm-usage', '--single-process'],
          devtools: false, headless: true, dumpio:true,
        },
      ],
```

**Describe the bug**

Unable to run tests in the terminal as some docker image issue compatiblity with M1 chip.
tried the following
1. Adding `FROM --platform=linux/amd64` to the Dockerfile
2. Adding arguments such as `--single-process`
3. Adding arguments such as `({ args: ['--no-sandbox', '--disable-setuid-sandbox', '--single-process'], devtools: false, headless: true, dumpio:true })`

Error logs:
```
  browserType.launch: Browser closed.
    ==================== Browser output: ====================
    [pid=322][err] [0127/130619.423543:FATAL:zygote_main_linux.cc(161)] Check failed: sandbox::ThreadHelpers::IsSingleThreaded().
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006af3f10 <unknown>
    [pid=322][err] #3 0x004006af4a5e <unknown>
    [pid=322][err] #4 0x004006680c57 <unknown>
    [pid=322][err] #5 0x00400667b7ae <unknown>
    [pid=322][err] #6 0x00400667d16b <unknown>
    [pid=322][err] #7 0x00400667a9c5 <unknown>
    [pid=322][err] #8 0x00400667b49e <unknown>
    [pid=322][err] #9 0x0040066d9b2b <unknown>
    [pid=322][err] #10 0x0040066d9893 <unknown>
    [pid=322][err] #11 0x0040066d8505 <unknown>
    [pid=322][err] #12 0x00400343b1f3 <unknown>
    [pid=322][err] #13 0x00400ef190b3 <unknown>
    [pid=322][err] #14 0x00400343b02a <unknown>
    [pid=322][err] Crash keys:
    [pid=322][err]   "switch-8" = "--allow-pre-commit-input"
    [pid=322][err]   "switch-7" = "--use-angle=swiftshader-webgl"
    [pid=322][err]   "switch-6" = "--use-gl=angle"
    [pid=322][err]   "switch-5" = "--ozone-platform=headless"
    [pid=322][err]   "switch-4" = "--headless"
    [pid=322][err]   "switch-3" = "--headless"
    [pid=322][err]   "switch-2" = "--no-sandbox"
    [pid=322][err]   "switch-1" = "--type=zygote"
    [pid=322][err]   "num-switches" = "8"
    [pid=322][err]
    [pid=322][err] [0127/130619.421174:FATAL:zygote_main_linux.cc(161)] Check failed: sandbox::ThreadHelpers::IsSingleThreaded().
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006af3f10 <unknown>
    [pid=322][err] #3 0x004006af4a5e <unknown>
    [pid=322][err] #4 0x004006680c57 <unknown>
    [pid=322][err] #5 0x00400667b7ae <unknown>
    [pid=322][err] #6 0x00400667d16b <unknown>
    [pid=322][err] #7 0x00400667a9c5 <unknown>
    [pid=322][err] #8 0x00400667b49e <unknown>
    [pid=322][err] #9 0x0040066d9b2b <unknown>
    [pid=322][err] #10 0x0040066d9893 <unknown>
    [pid=322][err] #11 0x0040066d8505 <unknown>
    [pid=322][err] #12 0x00400343b1f3 <unknown>
    [pid=322][err] #13 0x00400ef190b3 <unknown>
    [pid=322][err] #14 0x00400343b02a <unknown>
    [pid=322][err] Crash keys:
    [pid=322][err]   "switch-9" = "--allow-pre-commit-input"
    [pid=322][err]   "switch-8" = "--use-angle=swiftshader-webgl"
    [pid=322][err]   "switch-7" = "--use-gl=angle"
    [pid=322][err]   "switch-6" = "--ozone-platform=headless"
    [pid=322][err]   "switch-5" = "--headless"
    [pid=322][err]   "switch-4" = "--headless"
    [pid=322][err]   "switch-3" = "--no-sandbox"
    [pid=322][err]   "switch-2" = "--no-zygote-sandbox"
    [pid=322][err]   "switch-1" = "--type=zygote"
    [pid=322][err]   "num-switches" = "9"
    [pid=322][err]
    [pid=322][err] Received signal Received signal 66
    [pid=322][err]
    [pid=322][err] #0 0x004006b75f99 #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##44  0x0x00400ef3818b00400ef3818b  <unknown>
    [pid=322][err] <unknown>#
    [pid=322][err] 5 #50x 00400ef178590x 00400ef17859 <unknown><unknown>
    [pid=322][err] #
    [pid=322][err] 6 #0x6004006b74d05  0x004006b74d05 <unknown>
    [pid=322][err] #7 0x004006af433f <unknown>
    [pid=322][err] #7 0x004006af433f <unknown>
    [pid=322][err] #8 0x004006af4a5e <unknown>
    [pid=322][err] #8 0x004006af4a5e <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##99  0x0x004006680c57004006680c57  <unknown>
    [pid=322][err] #10 0x00400667b7ae <unknown>
    [pid=322][err] #10 0x00400667b7ae <unknown>
    [pid=322][err] #11 0x00400667d16b <unknown>
    [pid=322][err] #11 0x00400667d16b <unknown>
    [pid=322][err] #12 0x00400667a9c5 <unknown>
    [pid=322][err] #12 0x00400667a9c5 <unknown>
    [pid=322][err] #13 0x00400667b49e <unknown>
    [pid=322][err] #13 0x00400667b49e <unknown>
    [pid=322][err] #14 0x0040066d9b2b <unknown>
    [pid=322][err] #14 0x0040066d9b2b <unknown>
    [pid=322][err] #15 0x0040066d9893 <unknown>
    [pid=322][err] #15 0x0040066d9893 <unknown>
    [pid=322][err] #16 0x0040066d8505 <unknown>
    [pid=322][err] #16 0x0040066d8505 <unknown>
    [pid=322][err] #17 0x00400343b1f3 <unknown>
    [pid=322][err] #17 0x00400343b1f3 <unknown>
    [pid=322][err] #18 0x00400ef190b3 <unknown>
    [pid=322][err] #18 0x00400ef190b3 <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##1919  0x0x00400343b02a 00400343b02a <unknown>
    [pid=322][err]   r8: <unknown>
    [pid=322][err]   r8: 0000000000000000  r9: 000000400e198300 r10: 0000000000000008 r11: 000000400f0b2780
    [pid=322][err]  r12: 00003842001c4000 r13: 000000400e198560 r14: 00003842001c4010 r15: aaaaaaaaaaaaaaaa
    [pid=322][err]   di: 0000000000000002  si: 000000400e1983000000000000000000  bp: 000000400e198550  r9:   bx: 000000401027f900000000400e1982f0
    [pid=322][err]   dx:  r10: 00000000000000000000000000000008  ax:  r11: 0000000000000000000000400f0b2780  cx: 0000000000000027
    [pid=322][err]   sp: 000000400e198300
    [pid=322][err]  r12:   ip: 00003154001c4000000000400ef3818b r13:  efl: 000000400e1985500000000000000246 r14:  cgf: 00003154001c4010002b000000000033 r15:  erf: aaaaaaaaaaaaaaaa0000000000000000
    [pid=322][err]
    [pid=322][err]   di:  trp: 0000000000000002ffffffffffffffff msk:   si: 0000000000000000 cr2: 000000400e1982f00000000000000000  bp: 000000400e198540  bx: 000000401027f900
    [pid=322][err]
    [pid=322][err]   dx: 0000000000000000  ax: 0000000000000000  cx: 0000000000000027  sp: 000000400e1982f0[end of stack trace]
    [pid=322][err]
    [pid=322][err]   ip: 000000400ef3818b efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
    [pid=322][err]  trp: ffffffffffffffff msk: 0000000000000000 cr2: 0000000000000000
    [pid=322][err] [end of stack trace]
    [pid=322][err] qemu: uncaught target signal 6 (Aborted) - core dumped
    [pid=322][err] qemu: uncaught target signal 6 (Aborted) - core dumped
    [pid=322][err] Received signal 11 SEGV_MAPERR 12fb2d23fb1547
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown>
    [pid=322][err] #4 0x004007406a98 <unknown>
    [pid=322][err] #5 0x00400740723a <unknown>
    [pid=322][err] #6 0x00400741528c <unknown>
    [pid=322][err] #7 0x0040074151e2 <unknown>
    [pid=322][err] #8 0x00400740d7e1 <unknown>
    [pid=322][err] #9 0x004007403fe1 <unknown>
    [pid=322][err] #10 0x004007409aa6 <unknown>
    [pid=322][err] #11 0x004007409b3e <unknown>
    [pid=322][err] #12 0x0040073ff877 <unknown>
    [pid=322][err] #13 0x0040073ff7b7 <unknown>
    [pid=322][err] #14 0x00400740dbb1 <unknown>
    [pid=322][err] #15 0x0040073fec62 <unknown>
    [pid=322][err] #16 0x00400748e667 <unknown>
    [pid=322][err] #17 0x0040047159a8 <unknown>
    [pid=322][err] #18 0x00400bf996da <unknown>
    [pid=322][err] #19 0x00400667d649 <unknown>
    [pid=322][err] #20 0x00400667d157 <unknown>
    [pid=322][err] #21 0x00400667a9c5 <unknown>
    [pid=322][err] #22 0x00400667b49e <unknown>
    [pid=322][err] #23 0x0040066d99ce <unknown>
    [pid=322][err] #24 0x0040066d96fd <unknown>
    [pid=322][err] #25 0x00400343b1f3 <unknown>
    [pid=322][err] #26 0x00400ef190b3 <unknown>
    [pid=322][err] #27 0x00400343b02a <unknown>
    [pid=322][err]   r8: aaaaaaaaaaaaaaaa  r9: 000018900040c000 r10: ffffffffffe863c0 r11: 000000400f0b2940
    [pid=322][err]  r12: 00001890001f2740 r13: 0000000000000100 r14: 00001890001f27c0 r15: 0000000000000100
    [pid=322][err]   di: 00001890001e6cfa  si: 00001890001e6cf8  bp: 000000400e199130  bx: fb12fb2d23fb153f
    [pid=322][err]   dx: 0000000000000000  ax: 000000000000ffee  cx: 0000000000000000  sp: 000000400e1990a0
    [pid=322][err]   ip: 0000004007406a98 efl: 0000000000000286 cgf: 002b000000000033 erf: 0000000000000004
    [pid=322][err]  trp: ffffffffffffffff msk: 0000000000000000 cr2: 0012fb2d23fb1547
    [pid=322][err] [end of stack trace]
    [pid=322][err] qemu: uncaught target signal 11 (Segmentation fault) - core dumped
    =========================== logs ===========================
    <launching> /ms-playwright/chromium-930007/chrome-linux/chrome --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose,MediaRouter --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --disable-sync --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --no-service-autorun --headless --hide-scrollbars --mute-audio --blink-settings=primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4 --no-sandbox --user-data-dir=/tmp/playwright_chromiumdev_profile-qoF5EY --remote-debugging-pipe --no-startup-window
    <launched> pid=322
    [pid=322][err] [0127/130619.109708:ERROR:stack_trace_posix.cc(704)] Failed to parse the contents of /proc/self/maps
    [pid=322][err] [0127/130619.394440:ERROR:stack_trace_posix.cc(704)] Failed to parse the contents of /proc/self/maps
    [pid=322][err] [0127/130619.396637:ERROR:stack_trace_posix.cc(704)] Failed to parse the contents of /proc/self/maps
    [pid=322][err] [0127/130619.423543:FATAL:zygote_main_linux.cc(161)] Check failed: sandbox::ThreadHelpers::IsSingleThreaded().
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006af3f10 <unknown>
    [pid=322][err] #3 0x004006af4a5e <unknown>
    [pid=322][err] #4 0x004006680c57 <unknown>
    [pid=322][err] #5 0x00400667b7ae <unknown>
    [pid=322][err] #6 0x00400667d16b <unknown>
    [pid=322][err] #7 0x00400667a9c5 <unknown>
    [pid=322][err] #8 0x00400667b49e <unknown>
    [pid=322][err] #9 0x0040066d9b2b <unknown>
    [pid=322][err] #10 0x0040066d9893 <unknown>
    [pid=322][err] #11 0x0040066d8505 <unknown>
    [pid=322][err] #12 0x00400343b1f3 <unknown>
    [pid=322][err] #13 0x00400ef190b3 <unknown>
    [pid=322][err] #14 0x00400343b02a <unknown>
    [pid=322][err] Crash keys:
    [pid=322][err]   "switch-8" = "--allow-pre-commit-input"
    [pid=322][err]   "switch-7" = "--use-angle=swiftshader-webgl"
    [pid=322][err]   "switch-6" = "--use-gl=angle"
    [pid=322][err]   "switch-5" = "--ozone-platform=headless"
    [pid=322][err]   "switch-4" = "--headless"
    [pid=322][err]   "switch-3" = "--headless"
    [pid=322][err]   "switch-2" = "--no-sandbox"
    [pid=322][err]   "switch-1" = "--type=zygote"
    [pid=322][err]   "num-switches" = "8"
    [pid=322][err]
    [pid=322][err] [0127/130619.421174:FATAL:zygote_main_linux.cc(161)] Check failed: sandbox::ThreadHelpers::IsSingleThreaded().
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006af3f10 <unknown>
    [pid=322][err] #3 0x004006af4a5e <unknown>
    [pid=322][err] #4 0x004006680c57 <unknown>
    [pid=322][err] #5 0x00400667b7ae <unknown>
    [pid=322][err] #6 0x00400667d16b <unknown>
    [pid=322][err] #7 0x00400667a9c5 <unknown>
    [pid=322][err] #8 0x00400667b49e <unknown>
    [pid=322][err] #9 0x0040066d9b2b <unknown>
    [pid=322][err] #10 0x0040066d9893 <unknown>
    [pid=322][err] #11 0x0040066d8505 <unknown>
    [pid=322][err] #12 0x00400343b1f3 <unknown>
    [pid=322][err] #13 0x00400ef190b3 <unknown>
    [pid=322][err] #14 0x00400343b02a <unknown>
    [pid=322][err] Crash keys:
    [pid=322][err]   "switch-9" = "--allow-pre-commit-input"
    [pid=322][err]   "switch-8" = "--use-angle=swiftshader-webgl"
    [pid=322][err]   "switch-7" = "--use-gl=angle"
    [pid=322][err]   "switch-6" = "--ozone-platform=headless"
    [pid=322][err]   "switch-5" = "--headless"
    [pid=322][err]   "switch-4" = "--headless"
    [pid=322][err]   "switch-3" = "--no-sandbox"
    [pid=322][err]   "switch-2" = "--no-zygote-sandbox"
    [pid=322][err]   "switch-1" = "--type=zygote"
    [pid=322][err]   "num-switches" = "9"
    [pid=322][err]
    [pid=322][err] Received signal Received signal 66
    [pid=322][err]
    [pid=322][err] #0 0x004006b75f99 #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##44  0x0x00400ef3818b00400ef3818b  <unknown>
    [pid=322][err] <unknown>#
    [pid=322][err] 5 #50x 00400ef178590x 00400ef17859 <unknown><unknown>
    [pid=322][err] #
    [pid=322][err] 6 #0x6004006b74d05  0x004006b74d05 <unknown>
    [pid=322][err] #7 0x004006af433f <unknown>
    [pid=322][err] #7 0x004006af433f <unknown>
    [pid=322][err] #8 0x004006af4a5e <unknown>
    [pid=322][err] #8 0x004006af4a5e <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##99  0x0x004006680c57004006680c57  <unknown>
    [pid=322][err] #10 0x00400667b7ae <unknown>
    [pid=322][err] #10 0x00400667b7ae <unknown>
    [pid=322][err] #11 0x00400667d16b <unknown>
    [pid=322][err] #11 0x00400667d16b <unknown>
    [pid=322][err] #12 0x00400667a9c5 <unknown>
    [pid=322][err] #12 0x00400667a9c5 <unknown>
    [pid=322][err] #13 0x00400667b49e <unknown>
    [pid=322][err] #13 0x00400667b49e <unknown>
    [pid=322][err] #14 0x0040066d9b2b <unknown>
    [pid=322][err] #14 0x0040066d9b2b <unknown>
    [pid=322][err] #15 0x0040066d9893 <unknown>
    [pid=322][err] #15 0x0040066d9893 <unknown>
    [pid=322][err] #16 0x0040066d8505 <unknown>
    [pid=322][err] #16 0x0040066d8505 <unknown>
    [pid=322][err] #17 0x00400343b1f3 <unknown>
    [pid=322][err] #17 0x00400343b1f3 <unknown>
    [pid=322][err] #18 0x00400ef190b3 <unknown>
    [pid=322][err] #18 0x00400ef190b3 <unknown><unknown>
    [pid=322][err]
    [pid=322][err] ##1919  0x0x00400343b02a 00400343b02a <unknown>
    [pid=322][err]   r8: <unknown>
    [pid=322][err]   r8: 0000000000000000  r9: 000000400e198300 r10: 0000000000000008 r11: 000000400f0b2780
    [pid=322][err]  r12: 00003842001c4000 r13: 000000400e198560 r14: 00003842001c4010 r15: aaaaaaaaaaaaaaaa
    [pid=322][err]   di: 0000000000000002  si: 000000400e1983000000000000000000  bp: 000000400e198550  r9:   bx: 000000401027f900000000400e1982f0
    [pid=322][err]   dx:  r10: 00000000000000000000000000000008  ax:  r11: 0000000000000000000000400f0b2780  cx: 0000000000000027
    [pid=322][err]   sp: 000000400e198300
    [pid=322][err]  r12:   ip: 00003154001c4000000000400ef3818b r13:  efl: 000000400e1985500000000000000246 r14:  cgf: 00003154001c4010002b000000000033 r15:  erf: aaaaaaaaaaaaaaaa0000000000000000
    [pid=322][err]
    [pid=322][err]   di:  trp: 0000000000000002ffffffffffffffff msk:   si: 0000000000000000 cr2: 000000400e1982f00000000000000000  bp: 000000400e198540  bx: 000000401027f900
    [pid=322][err]
    [pid=322][err]   dx: 0000000000000000  ax: 0000000000000000  cx: 0000000000000027  sp: 000000400e1982f0[end of stack trace]
    [pid=322][err]
    [pid=322][err]   ip: 000000400ef3818b efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
    [pid=322][err]  trp: ffffffffffffffff msk: 0000000000000000 cr2: 0000000000000000
    [pid=322][err] [end of stack trace]
    [pid=322][err] qemu: uncaught target signal 6 (Aborted) - core dumped
    [pid=322][err] qemu: uncaught target signal 6 (Aborted) - core dumped
    [pid=322][err] Received signal 11 SEGV_MAPERR 12fb2d23fb1547
    [pid=322][err] #0 0x004006b75f99 <unknown>
    [pid=322][err] #1 0x004006ae0513 <unknown>
    [pid=322][err] #2 0x004006b75a71 <unknown>
    [pid=322][err] #3 0x00400e1f53c0 <unknown>
    [pid=322][err] #4 0x004007406a98 <unknown>
    [pid=322][err] #5 0x00400740723a <unknown>
    [pid=322][err] #6 0x00400741528c <unknown>
    [pid=322][err] #7 0x0040074151e2 <unknown>
    [pid=322][err] #8 0x00400740d7e1 <unknown>
    [pid=322][err] #9 0x004007403fe1 <unknown>
    [pid=322][err] #10 0x004007409aa6 <unknown>
    [pid=322][err] #11 0x004007409b3e <unknown>
    [pid=322][err] #12 0x0040073ff877 <unknown>
    [pid=322][err] #13 0x0040073ff7b7 <unknown>
    [pid=322][err] #14 0x00400740dbb1 <unknown>
    [pid=322][err] #15 0x0040073fec62 <unknown>
    [pid=322][err] #16 0x00400748e667 <unknown>
    [pid=322][err] #17 0x0040047159a8 <unknown>
    [pid=322][err] #18 0x00400bf996da <unknown>
    [pid=322][err] #19 0x00400667d649 <unknown>
    [pid=322][err] #20 0x00400667d157 <unknown>
    [pid=322][err] #21 0x00400667a9c5 <unknown>
    [pid=322][err] #22 0x00400667b49e <unknown>
    [pid=322][err] #23 0x0040066d99ce <unknown>
    [pid=322][err] #24 0x0040066d96fd <unknown>
    [pid=322][err] #25 0x00400343b1f3 <unknown>
    [pid=322][err] #26 0x00400ef190b3 <unknown>
    [pid=322][err] #27 0x00400343b02a <unknown>
    [pid=322][err]   r8: aaaaaaaaaaaaaaaa  r9: 000018900040c000 r10: ffffffffffe863c0 r11: 000000400f0b2940
    [pid=322][err]  r12: 00001890001f2740 r13: 0000000000000100 r14: 00001890001f27c0 r15: 0000000000000100
    [pid=322][err]   di: 00001890001e6cfa  si: 00001890001e6cf8  bp: 000000400e199130  bx: fb12fb2d23fb153f
    [pid=322][err]   dx: 0000000000000000  ax: 000000000000ffee  cx: 0000000000000000  sp: 000000400e1990a0
    [pid=322][err]   ip: 0000004007406a98 efl: 0000000000000286 cgf: 002b000000000033 erf: 0000000000000004
    [pid=322][err]  trp: ffffffffffffffff msk: 0000000000000000 cr2: 0012fb2d23fb1547
    [pid=322][err] [end of stack trace]
    [pid=322][err] qemu: uncaught target signal 11 (Segmentation fault) - core dumped
    ============================================================

      at getBrowserPerProcess (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:19:39)
      at PlaywrightEnvironment.setup (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:143:48)
```

**Comment by aslushnikov** [comment]
This has been fixed in Playwright 1.17.0. Make sure to use a `mcr.microsoft.com/playwright:v1.17.0` or higher as well

**Comment by chanagonda** [comment]
> This has been fixed in Playwright 1.17.0. Make sure to use a `mcr.microsoft.com/playwright:v1.17.0` or higher as well

Thank you @aslushnikov , Its working

**Comment by zhanghuibin2013** [comment]
my version is 1.39.0，and i see the same thing. i don't known why 

> This has been fixed in Playwright 1.17.0. Make sure to use a `mcr.microsoft.com/playwright:v1.17.0` or higher as well
